config/common/errors.go

```diff
@@ -46,8 +46,9 @@ var (
 	ErrTooManySystemdSources = errors.New("only one of the following can be set: contents, contents_local")
 
 	// mount units
-	ErrMountUnitNoPath   = errors.New("path is required if with_mount_unit is true and format is not swap")
-	ErrMountUnitNoFormat = errors.New("format is required if with_mount_unit is true")
+	ErrMountUnitNoPath     = errors.New("path is required if with_mount_unit is true and format is not swap")
+	ErrMountUnitNoFormat   = errors.New("format is required if with_mount_unit is true")
+	ErrMountPointForbidden = errors.New("mount point is forbidden by fcos, use '/etc','/var' or their subdirectories ")
```

Comment:
Hmm, what about "forbidden mount point. Use a mount point that starts with `/etc` or `/var`".